Arch/README.md

```diff
@@ -0,0 +1,30 @@
+# Installation Guide
+## VirtualBox
+* Download [virtualBox](https://www.virtualbox.org/wiki/Downloads) recommended video for [help](https://www.youtube.com/watch?v=nwjZWHou8u0&t=196s) 
+*  Download [ISO image of arch linux](https://geo.mirror.pkgbuild.com/iso/2024.03.01/) then select this link
+![](imgs/archlink.img.png)
+**After downloading do the following steps to prepare your virtual box** 
+* select **new** option
+![](imgs/firstStep.img.png)
+* Write name of Distor like **Archlinux**
+![](imgs/distorname.img.png)
+* click here then choose **Other**
+![](imgs/browseISO.img.png)
+* choose IOS file then click **Open**
+![](imgs/selectISO.img.png)
+* click **Next**
+![](imgs/nextoption.img.png)
+* make **memory 2 MB** and **Processor 2 CPU **then click **Next** 
+![](imgs/setMemory&CPU.img.png)
+* click **Next**
+![](imgs/nextoption2.img.png)
+* click **Finish**
+![](imgs/finishoption.img.png)
+* Click **Start** 
+![](imgs/getstart.img.png)
+* choose the first option
+![](imgs/firstpage.img.png)
+* then this page will apper 
+![](imgs/secondpage.img.png)
+* **Now you should set up Arch Linux using this [repo](https://github.com/SalmaAlassal/dotfiles/blob/main/arch.md)**
```

Comment:
We could create another section over here and title it "Setting up Arch Linux"

This repo is an awesome resource. I think it's a good idea to redirect to it, instead of repeating the steps over 

A link to the Arch wiki would be great here too!